cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -200,8 +201,11 @@ def service_request(appliance, ansible_catalog_item):
 def service(appliance, ansible_catalog_item):
     _service = MyService(appliance, ansible_catalog_item.name)
     yield _service
-
-    _service.delete_if_exists()
+    try:
+        if _service.exists:
```

Comment:
This is a sentaku ContextualProperty, it did not have the method `delete_if_exists()`. But this was throwing `ItemNotFound`, so wrapped in a Try/Except